Src/java/cql-to-elm/src/test/java/org/cqframework/cql/cql2elm/TranslationTests.java

```diff
@@ -467,4 +467,38 @@ void abstractClassNotRetrievable() throws IOException {
                 errors.stream().map(Throwable::getMessage).collect(Collectors.toList());
         assertThat(errorMessages, contains("Specified data type DomainResource does not support retrieval."));
     }
+
+    @Test
+    void mappingExpansionsRespectSignatureLevel() throws IOException {
```

Comment:
Please add a test for Signature.None to contrast with the test below.  Also, it might be good to test other non-Overloads, non-None Signature types as well.